experimental/dds/tree/src/id-compressor/IdCompressor.ts

```diff
@@ -1624,11 +1621,8 @@ export class IdCompressor {
 		let serializedLocalState: SerializedLocalState | undefined;
 		if (hasSession) {
 			assert(newSessionIdMaybe === undefined && attributionInfoMaybe === undefined);
-			// TODO: This cast can be removed on typescript 4.6
-			[localSessionId, attributionInfo] =
-				serialized.sessions[(serialized as SerializedIdCompressorWithOngoingSession).localSessionIndex];
-			// TODO: This cast can be removed on typescript 4.6
-			serializedLocalState = (serialized as SerializedIdCompressorWithOngoingSession).localState;
+			[localSessionId, attributionInfo] = serialized.sessions[serialized.localSessionIndex];
+			serializedLocalState = serialized.localState;
```

Comment:
nit: was the comment regarding TS 4.6 just wrong? Just curious.

Reply:
This code was back ported from typescript 4.6 so its author knew it would be fixed by updating to 4.6, and didn't bother to figure out which minor version was needed. So the comment was correct, just not very specific.